statsd/receiver.go

```diff
@@ -149,24 +151,16 @@ func (mr *MetricReceiver) handleMessage(addr net.Addr, msg []byte) {
 		if len(line) > 1 {
 			metric, err := mr.parseLine(line)
 			if err != nil {
-				log.Errorf("Error parsing line %q from %s: %v", line, addr, err)
+				log.Warnf("Error parsing line %q from %s: %v", line, addr, err)
```

Comment:
I changed some errors to be warn messages - if someone sends invalid data and we cannot parse it, it is not an error from the application point of view. IMHO error is something like "Cannot open socket to accept connections"/etc.